### ISSUE TITLE
Blank screen

### ISSUE THREAD

**Issue by stevelitchfield** [kypeli/Podcatcher#41]
In v2.4, if you pause a podcast with Podcatcher in the foreground and then let the screen timeout, when the lock screen is dismissed, the app's display is blank and unusable. Or, if playback was started from a podcast's episode list then you just get a Settings icon and nothing else.

This is a new bug and wasn't there in previous versions.

**Comment by kypeli** [comment]
I tried this with my Lumia 800 with WP 7.8 but I can't seem to be able to reproduce this. What I did was:
- Start playback from play queue view from the toolbar icon. 
- Podcast is now playing, I am in the play queue view.
- I press the Pause button in the toolbar. Podcast is now paused. 
- I let the screen dim, screen lock gets on. I unlock the screen.
- UI shows up.

I also tried this by starting the podcast and pausing the playback from the lock screen, letting the screen dim and then unlocking. I did this also from the episode listings view. I couldn't no reproduce it. 

So it might be a WP8 issue or I am just doing it wrong. Can you please elaborate a bit on the steps?

**Comment by stevelitchfield** [comment]
Start Podcatcher
Tap on the top item in 'Play history'
(Oddly), the episode list is shown (I'd expect it to go to a single episode playback screen here?)
Start an episode playing
Pause it
Wait for the screen to time out
Unlock the screen
----> Screen is now blank apart from a lonely settings icon at the bottom

**Comment by kypeli** [comment]
I still can't reproduce this (with Lumia 800 / WP 7.8) :(

So I am in the player view when the screen dims and that's what the steps above would suggest I should be in, right? But for me the outcome is the same as in bug #21. 

Based on the comment about the settings icon in the bottom, so I also went back to the episodes list after pausing the episode, but also in this case I got back to the episodes list after resuming. 

But I will try later on with my Lumia 920. :)
